.github/workflows/build.yml

```diff
@@ -21,5 +21,25 @@ jobs:
           java-version: 11
           distribution: 'zulu'
           cache: 'gradle'
+      - name: Extract repository name
+        id: extract_repo_name
+        run: echo ::set-output name=REPO::${GITHUB_REPOSITORY##*/}
       - name: Build
-        run: ./gradlew build --stacktrace
+        run: ./gradlew build --stacktrace ${{ steps.extract_repo_name.outputs.REPO }}-web:installDist"
+      - name: Build slim container image
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: Dockerfile.slim
+          push: true
+          tags: |
+            "${{ env.CONTAINER_REGISTRY }}/${{ steps.extract_repo_name.outputs.REPO }}:spinnaker-${{ env.GITHUB_REF_NAME }}-latest-unvalidated"
```

Comment:
Currently:
1. all images also have a tag with `{date}` in it, eg: `spinnaker-release-1.25.x-20211208040018-ubuntu`
2. sometimes the same image (SHA) is tagged with different dates, eg: SHA `0e633dcfe<snip>` is tagged `spinnaker-release-1.25.x-20211208040018-ubuntu` and `spinnaker-release-1.25.x-20211214191852-ubuntu` amongst others.
3. it's not clear to me what timezone the date is in. This would be more obvious when GHA is doing pushes as the Action steps are publicly visible
4. having the date does give a kind of "immutable" feel, though technically there is nothing to limit an overwrite push

We could:
a) remove the `{date}` then the `:*latest*` image is the latest and preceding images will be overwritten (different SHA though).
b) consider using a short or long `SHA` instead of date to preserve individual merge commit images?

[Container registries list push date](https://console.cloud.google.com/artifacts/docker/spinnaker-community/us/docker/rosco) so it's possible to understand when the image was pushed.

Reply:
I don't see `{date}` here anywhere.  Does `build-push-action` add it?  I like a git sha more than a date since it links back to the code.  But I also like a date because it helps give a natural sort order.  So my current reflex is to have both...unless we somehow tag commits with the docker tag, and then we don't need the git sha in the docker tag.

Reply:
Sorry the existing CI seems to add a `{date}` and that can be seen in https://console.cloud.google.com/artifacts/docker/spinnaker-community/us/docker/rosco.

I've added both git commit and date time to tag variations in https://github.com/spinnaker/rosco/pull/841/commits/e47dd63fb60df27701ac0b3d993a5b71afe27280

[Dockerfile's](https://github.com/spinnaker/rosco/blob/master/Dockerfile.slim#L9 ) contain non-idempotent commands like `apk update` so a single git commit could be built multiple times resulting in a different container image SHA.
If we don't have date time then we will be overwriting previous images.

Overall we might be over indexing on tags but we could pare it back later once we shake out the complete process?